### DISCUSSION TITLE
Sobol

### DISCUSSION THREAD

**Discussion by matukumilli** [SSCHAcode/python-sscha#107] [Q&A] [answered]
What is this sobol algorithm? Could anyone pls provide some links to know about it?
Curious to know how it can help reduce no. of configurations.

Warm regards,
Prasad Matukumilli
JNCASR, Bangalore.

**Comment by diegomartinez2** [comment]
Hi a fast answer will be to check [wikipedia](https://en.wikipedia.org/wiki/Sobol_sequence). But I'll recommend to take a look at this answers from [math.stackexchange](https://math.stackexchange.com/questions/888490/understanding-sobol-sequences).

**Reply by diegomartinez2** [reply, marked as answer]
Sobol sequences reduces the number of configurations by doing a better mapping of the gaussian than a random distribution. By uniformity spreading the samplings with a low discrepancy sequence like Sobol it is possible to reduce the number of configurations needed.  Low discrepancy sequences tend to sample space "more uniformly" than random numbers. Algorithms that use such sequences may have superior convergence.

**Reply by matukumilli** [reply]
Thanks, Diego! That makes sense. I was thinking about having a more flexible mode-wise displacement pattern could be a way forward. For e.g. displacements due to FE and AFD modes in SrTiO3 would distribute in a pattern different from other modes. 

I look forward for your comments

Regards,
Prasad

**Reply by mesonepigreco** [reply]
We are thinking to ways to improve the convergence with the number of configurations. We find that the Sobol sampling slightly reduces the number of displacements to be sampled. It is possible that the use of some special point could be beneficial, however it is not trivial, as we are already exploiting a lot of tricks to reduce the stochastic noise.
